renderer/src/pages/Onboarding.tsx

```diff
@@ -24,9 +24,6 @@ const OnboardingPage = (): JSX.Element => {
   useEffect(() => {
     (async () => {
       await sleep(2000)
-      if (await getFilAddress()) {
-        return navigate('/dashboard', { replace: true })
-      }
```

Comment:
isn't this redirect missing now? I don't see any other code redirecting to that url

Reply:
This is addressed on #334